pkg/standalone/client.go

```diff
@@ -0,0 +1,32 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+type DaprProcess interface {
+	List() ([]ListOutput, error)
+}
+
+type daprProcess struct {
+}
+
+type Client interface {
```

Comment:
Can you please add documentation for this interface? BTW, if you can add Done, and resolve it would make it easier for reviewer to see the new changes.

Otherwise if you object to a review feedback you can also object but don't resolve, that way we can see the on-going conversation.

Reply:
Sorry, commented on an outdated diff.